src/guides/v2.3/frontend-dev-guide/themes/theme-structure.md

```diff
@@ -197,7 +197,7 @@ The directories and files structure described below is the most extended one. It
         optional
       </td>
       <td colspan="1">
-        Theme fonts.
+        Theme fonts and customized icons will be added here. `Theme-Icons`.
```

Comment:
What does `Theme-Icons` mean here?

Reply:
means Theme icon fonts.

Reply:
It seems strange to just have it sitting there alone. Let's remove it.

Reply:
This entire table is inconsistent, but just to keep clean, let's not use "will be" and make a clear statement, like "Contains theme fonts and customized icons."